src/setupTests.js

```diff
@@ -1,33 +1,23 @@
-import React from 'react'
-import { act } from 'react-dom/test-utils'
 import Adapter from 'enzyme-adapter-react-16'
-import { configure, mount } from 'enzyme'
+import { configure } from 'enzyme'
 import 'jest-localstorage-mock'
+import './store'
+import { act } from 'react-dom/test-utils'
+import { initialize } from './App'
 
-import App, { initialized } from './App'
+/*******************************************************
+ *    Global setup that runs before each test file     *
+ * ****************************************************/
```

Comment:
When a former developer created `setupTests`, all of the tests used `mount`. Now that we only have a couple mount tests, I'm thinking all the logic in this file should actually belong in `createTestApp`. We don't want to `initialize` the app for non-mount tests. Does that seem right?

Reply:
Yes that makes sense. But we may need to `initialize` db for tests that use `store.dispatch` because it calls local db sync. So we should probably stop local db sync unless we need it. Is there any available way we can start application with local sync behavior set to desired state ?

Reply:
> But we may need to `initialize` db for tests that use `store.dispatch` because it calls local db sync. 

I'm not sure I follow. What calls local db sync? And what is the connection between `store.dispatch` and initialize?

I do recognize that the db needs to be initialized for any tests that use the db though.

> So we should probably stop local db sync unless we need it. Is there any available way we can start application with local sync behavior set to desired state ?

By "stop" do you mean cancel something that has been started, or avoid starting it to begin with? Which part of the local db sync are you trying to cancel exactly? Is it the dispatch of [`loadLocalState` in App.js](https://github.com/cybersemics/em/blob/dev/src/App.js#L36)? If not, maybe you can post a link to the line number.

Reply:
> By "stop" do you mean cancel something that has been started, or avoid starting it to begin with? Which part of the local db sync are you trying to cancel exactly? Is it the dispatch of [`loadLocalState` in App.js](https://github.com/cybersemics/em/blob/dev/src/App.js#L36)? If not, maybe you can post a link to the line number

I meant after `store.dispatch` is called with `thoughtIndex` and `contextIndex` updates then `syncQueue` will be flushed which will update data in local indexed db. Some test like shortcuts use global `store` unlike reducers tests. So if db is not initialized it will throw errors. So for such tests we must prevent sync. Does this make sense ?

Reply:
Yes, that makes sense. I will create a separate issue for this refactor. Use `store` as-is/as-needed in this PR. Thanks!

Reply:
#781